### PR TITLE
Add link to plotly-notebook-js

### PR DESCRIPTION
**Pull request by mdtusz**
Once we have a "getting started" page in the docs, we can add that as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,7 +128,8 @@ Open-source clients to the plotly.js APIs are available at these links:
 |**R / RStudio**| [ropensci/plotly](https://github.com/ropensci/plotly) | [plot.ly/r/getting-started](https://plot.ly/r/getting-started) |
 |**Python / Pandas / IPython notebook**| [plotly/plotly.py](https://github.com/plotly/plotly.py) | [plot.ly/python/getting-started](https://plot.ly/python/getting-started) |
 |**MATLAB**| [plotly/matlab-api](https://github.com/plotly/matlab-api) | [plot.ly/matlab/getting-started](https://plot.ly/matlab/getting-started) |
-|**node.js**| [plotly/plotly-nodejs](https://github.com/plotly/plotly-nodejs) | [plot.ly/nodejs/getting-started](https://plot.ly/nodejs/getting-started) |
+|**node.js / Tonicdev / Jupyter notebook **| [plotly/plotly-notebook-js](https://github.com/plotly/plotly-notebook-js) | |
```
Comment (etpinard) [inline README.md]: Hmm. We have *Python / Pandas / IPython notebook* two lines above. I'd put *Tonicdev* only on this line to avoid confusion.

Comment (etpinard) [inline README.md]: also

image

bring those `*`s together.

**Comment by mdtusz** [comment]
Do you mean to remove "Jupyter"? I'd like to keep that in for searchability - lots of people know Jupyter but Tonicdev is still relatively new on the scene.

**Comment by cldougl** [comment]
+1 for leaving Jupyter notebook -- I think that could draw more interest

**Comment by mdtusz** [comment]
Is this ok to merge and close?

**Comment by etpinard** [comment]
:dancer:

**Pull request merged**